File: src/main/kotlin/io/github/dqualizer/dqlang/types/dam/architecture/apischema/field/Input.java
package io.github.dqualizer.dqlang.types.dam.architecture.apischema.field;

import lombok.*;

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Data
public class Input {

    private String name;
    private String in;
    private boolean required;
    private String type;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/openapiadapter/dqlang/field/Input.kt
package io.github.dqualizer.dqlang.archive.openapiadapter.dqlang.field

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.ToString

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
class Input {
    val name: String? = null
    val `in`: String? = null
    val required = false
    val type: String? = null
}
